src/Orleans.Runtime/MembershipService/ClusterMember.cs

```diff
@@ -5,21 +5,26 @@ namespace Orleans.Runtime
     [Serializable]
     public sealed class ClusterMember : IEquatable<ClusterMember>
     {
-        public ClusterMember(SiloAddress siloAddress, SiloStatus status)
+        public ClusterMember(SiloAddress siloAddress, SiloStatus status, string name)
         {
             this.SiloAddress = siloAddress ?? throw new ArgumentNullException(nameof(siloAddress));
             this.Status = status;
+            this.Name = name;
         }
 
         public SiloAddress SiloAddress { get; }
         public SiloStatus Status { get; }
+        public string Name { get; }
```

Comment:
This will be a breaking change for 3.3, no?

Reply:
Shouldn't be, since this isn't sent over the wire

Reply:
Gotcha, you get the name from `MembershipEntry`